docs/dynamo.md

```diff
@@ -23,28 +23,28 @@ def eval_model(loader):
 ```
 > **NOTE:** inference backend name `torchxla_trace_once` is subject to change.
 
-With the `torch.compile` you will see that PyTorch/XLA only traces the resent18 model once during the init time and executes the compiled binary everytime `dynamo_resnet18` is invoked, instead of tracing the model every time. Note that currently Dynamo does not support fallback so if there is an op that can not be traced by XLA, it will error out. Feel free to open a github issue and we We prioritize adding the lowering. We will add the fallback support for dynamo in the upcoming 2.1 release. Here is a inference speed analysis to compare Dynamo and Lazy using torch bench on Cloud TPU v4-8
-
-| model | Speed up |
-| --- | ----------- |
-resnet18 | 1.768
-resnet50 | 1.61
-resnext50_32x4d	| 1.328
-alexnet | 1.261
-mobilenet_v2 | 2.017
-mnasnet1_0 | 1.686
-vgg16 | 1.155
-BERT_pytorch | 3.502
-squeezenet1_1 | 1.674
-timm_vision_transformer | 3.138
-average | 1.9139
+With the `torch.compile` you will see that PyTorch/XLA only traces the resent18 model once during the init time and executes the compiled binary every time `dynamo_resnet18` is invoked, instead of tracing the model every time. Note that currently Dynamo does not support fallback so if there is an op that can not be traced by XLA, it will error out. Feel free to open a github issue and we We prioritize adding the lowering. We will add the fallback support for dynamo in the upcoming 2.1 release. Here is a inference speed analysis to compare Dynamo and Lazy using torch bench on Cloud TPU v4-8
+
+| model                   | Speed up |
+| ----------------------- | -------- |
```

Comment:
in the rendered version I think it is being displayed correctly?

Reply:
this change was made automatically by vscode, but i have reverted it since there is no difference in the display